sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -23,9 +23,16 @@
     <SupportedPlatform Include="browser" />
   </ItemGroup>
 
+  <!-- Upgrade System.Diagnostics.DiagnosticSource to 6.0.1 in netcoreapp3.1+ -->
```

Comment:
Given we are overriding the older version I'd not include the 6.0.1 version in the comment as that will get outdated as soon as we update the shared version.